### PR TITLE
Fix [footway][sac_scale] color and [unpaved_*] rendered as footways

### PR DESCRIPTION
**Pull request by pastk**
1. In the main style `highway=footway sac_scale=*` was rendered as a white footway. Fixed the color to be same as `highway=path` (mentioned in #6658).

2. In vehicle style e.g. `highway-path psurface-unpaved_bad` appeared at z14- as white dash while ordinary highway-paths are z17-. Removed rendering of these types. 
See 2nd and 3rd screenshots in https://github.com/organicmaps/organicmaps/pull/6590#issuecomment-1821222179 and a dark screenshot in https://github.com/organicmaps/organicmaps/pull/6590#issuecomment-1824213963

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Thanks! LGTM

cc @vng

**Review by vng — APPROVED** [review]
Should we add some todo for [psurface=unpaved_bad] here?
I suppose, it was an attempt to highlight unpaved car roads, but not paths.

**Comment by pastk** [comment]
> Should we add some todo for [psurface=unpaved_bad] here? I suppose, it was an attempt to highlight unpaved car roads, but not paths.

Its discussed in 
- https://github.com/organicmaps/organicmaps/issues/5967

I'll add an update there.

File: data/styles/clear/include/Roads.mapcss [hunk 1]
```diff
@@ -608,14 +608,14 @@ line|z14-[highway=raceway],
...
-line|z14-[highway=path],
-line|z14-[highway=footway][sac_scale],
-{color: @path;opacity: 1;}
 line|z14-[highway=bridleway]
 {color: @bridleway;opacity: 1;}
 line|z15-[highway=footway],
 line|z15-[highway=steps]
 {color: @footway;opacity: 1;}
+line|z14-[highway=path],
+line|z14-[highway=footway][sac_scale],
```
[the remark this answers is not in the document]
Comment (pastk) [reply data/styles/clear/include/Roads.mapcss:617]: I can add a comment, sure.
The problem is we have many similar cases when such kind of inheritance leads to unobvious bugs...

**Comment by pastk** [comment]
Added a comment and rebased.

**Pull request merged**